ejcIMC.py

```diff
@@ -25,6 +25,25 @@ def imcCalculate(statureValue, weigthValue):
     IMC = (weigthValue/(statureValue**2))
     return IMC
 
+def yourImc(imc):
+    valueImc = imc
+    if imc <= 16:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez severa')
+    elif imc <= 16.99:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez moderada')
+    elif imc <= 18.49:
+        print('Su indice de masa corporal es ',imc ,': infrapeso e indica que se encuentra en delgadez aceptable')
+    elif imc <= 24.99:
+        print('Su indice de masa corporal es ',imc ,': pso normal e indica que se encuentra en con un peso adecuado')
+    elif imc <= 29.99:
+        print('Su indice de masa corporal es ',imc ,': sobre peso e indica que se encuentra en en un peso un poco encima de lo normal')
+    elif imc <= 34.99:
+        print('Su indice de masa corporal es ',imc ,': obeso e indica que se encuentra en el primer grado de obesidad')
+    elif imc <= 40.00:
+        print('Su indice de masa corporal es ',imc ,': obeso e indica que se encuentra en el segundo gado de obesidad')
```

Comment:
revise the print message, it say 'en el segundo gado de...', correct the "gado"'s word